packages/react/src/components/DatePicker/DatePicker.test.tsx

```diff
@@ -410,195 +370,117 @@ describe('DatePicker', () => {
       },
     );
   });
+});
 
-  describe('when Calendar properties are not specified', () => {
-    const datePicker = shallow(<DatePickerBase />);
-    datePicker
-      .find(TextField)
-      ?.props()
-      .onClick?.({} as any);
-    const calendarProps = datePicker.find(Calendar).props();
-
-    it('renders Calendar with isMonthPickerVisible as true by defaut', () => {
-      expect(calendarProps.isMonthPickerVisible).toBe(true);
-    });
-
-    it('renders Calendar with showMonthPickerAsOverlay as false by defaut', () => {
-      expect(calendarProps.showMonthPickerAsOverlay).toBe(false);
-    });
-
-    it('renders Calendar with highlightCurrentMonth as false by defaut', () => {
-      expect(calendarProps.highlightCurrentMonth).toBe(false);
-    });
-
-    it('renders Calendar with showWeekNumbers as false by defaut', () => {
-      expect(calendarProps.showWeekNumbers).toBe(false);
-    });
-
-    it('renders Calendar with firstWeekOfYear as FirstWeekOfYear.FirstDay by defaut', () => {
-      expect(calendarProps.firstWeekOfYear).toBe(FirstWeekOfYear.FirstDay);
-    });
-
-    it('renders Calendar with showGoToToday as true by defaut', () => {
-      expect(calendarProps.showGoToToday).toBe(true);
-    });
-  });
-
-  describe('when Calendar properties are specified', () => {
-    const value = new Date(2017, 10, 1);
-    const today = new Date(2017, 9, 31);
-
-    const dateTimeFormatter: IDatePickerProps['dateTimeFormatter'] = {
-      formatMonthDayYear: () => 'm/d/y',
-      formatMonthYear: () => 'm/y',
-      formatDay: () => 'd',
-      formatMonth: () => 'm',
-      formatYear: () => 'y',
-    };
-
-    const datePicker = shallow(
+describe('DatePickerBase - custom calendar properties', () => {
+  const value = new Date(2017, 10, 1);
+  const today = new Date(2017, 9, 31);
+  const dateTimeFormatter = {
+    formatMonthDayYear: () => 'm/d/y',
+    formatMonthYear: () => 'm/y',
+    formatDay: () => 'd',
+    formatMonth: () => 'm',
+    formatYear: () => 'y',
+  };
+
+  it('renders Calendar with provided props and custom formatter', () => {
+    const { container } = render(
       <DatePickerBase
         isMonthPickerVisible={false}
-        showMonthPickerAsOverlay={true}
+        showMonthPickerAsOverlay
         value={value}
         today={today}
         firstDayOfWeek={2}
-        highlightCurrentMonth={true}
-        showWeekNumbers={true}
+        highlightCurrentMonth
+        showWeekNumbers
         firstWeekOfYear={FirstWeekOfYear.FirstFullWeek}
         showGoToToday={false}
         dateTimeFormatter={dateTimeFormatter}
       />,
     );
-    datePicker
-      .find(TextField)
-      ?.props()
-      .onClick?.({} as any);
-
-    const calendarProps = datePicker.find(Calendar).props();
+    const input = screen.getByRole('combobox');
+    userEvent.click(input);
 
-    it('renders Calendar with same isMonthPickerVisible', () => {
```

Comment:
this is incorrect, when `shallow` is used the good approach is to use `react-test-renderer` which give use access to these low level assertions. lets keep it that way. the `toMatchSnapshot` provides unnecessary noise which doesn't add much value.